packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -149,3 +149,16 @@ export function processAdditionalValidOrigins(validMessageOrigins: string[]): vo
   });
   GlobalVars.additionalValidOrigins = combinedOriginUrls;
 }
+
+/**
+ * @hidden
+ * Check if apiVersion developer sends follows the pattern starting with a lowercase 'v' and it is then
+ * followed by one or more digits only. If yes, return true. Otherwise, return false.
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function isFollowApiVersionLabelFormat(apiVersion: string): boolean {
```

Comment:
It would be nice if you can add unit tests for all the Util files you introduced for code coverage purposes.

Reply:
Test case added.

I don't think rest of util files need to add cases because they are just moved there from part of original APIs. In the other word, the original test cases have already covered them.